### PR TITLE
FCREPO-2473

### PR DESCRIPTION
**Pull request by jwestgard**
- works with the config file of the current import-export tool
- implements configurable containment predicates 
- containment predicates feature needs some more work

### CODE REVIEW

**Comment by jwestgard** [comment]
@dbernstein I wanted to get this submitted in case there are other tickets that require the configuration parsing piece, so this code can be developed against.  I will try to get the configurable membership predicates piece working and add another commit.

File: fcrepo_verify/resources.py [hunk 1]
```diff
@@ -94,6 +94,7 @@ def __init__(self, inputpath, config, logger):
         self.relpath = self.origpath[len(self.data_dir):]
         urlinfo = urlparse(config.repo)
         config_repo = urlinfo.scheme + "://" + urlinfo.netloc
+        self.type = "unkown"
```
Comment (dbernstein) [inline fcrepo_verify/resources.py]: Typo - should be "unknown"

**Comment by dbernstein** [comment]
Just a reminder:  if possible add  the Resolves: https://jira.duraspace.org/browse/FCREPO-2473 to bottom of the commit message.  I'll add it on th squash and merge

**Comment by jwestgard** [comment]
OK.  I had been doing that, but this ticket (specifically where it says, "act appropriately on all options") entails a bit more than what is captured in that PR.  I created a couple tickets yesterday to capture some of the options, but I think we need to fully document the import-export tool's behaviors and create tickets for any behaviors that aren't covered yet.

**Pull request merged**